scss/utilities/_deprecated.scss

```diff
@@ -0,0 +1,135 @@
+/*!
+ * Deprecated Utilities class
+ * Since 2.0.2
```

Comment:
@tinhvqbk You can put messages inside your css rulesets like so:

```
.center {
  text-align: "center";
  --deprecated: "WARNING: Use .other-class instead";
}
```
This way developers and users can see the message directly in their browsers and take action

Reply:
Ok

Reply:
added on https://github.com/gotitinc/aha-css/commit/04423e3429e8aeb83bf434e04838a40573f38f7f @namdaoduy